tests/integration_tests_control_files/euler_entropy_conservation/euler_entropy_conserving_split_forms_check.prm

```diff
@@ -14,20 +14,19 @@ set use_collocated_nodes = true
 
 set flux_reconstruction = cDG
 set use_inverse_mass_on_the_fly = true
-# set use_collocated_nodes = true
 
 # Note: this was added to turn off check_same_coords() -- has no other function when dim!=1
 set use_periodic_bc = true
 
 # numerical fluxes
 set conv_num_flux = two_point_flux
-set two_point_num_flux_type = IR 
+set two_point_num_flux_type = IR  # this is changed by the test
```

Comment:
why even pass one then? can remove this line

Reply:
Removed and replaced with a comment.